superset/assets/visualizations/chord.jsx

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-param-reassign */
+import d3 from 'd3';
+import { category21 } from '../javascripts/modules/colors';
+import './chord.css';
+
+function chordViz(slice, json) {
+  slice.container.html('');
+
+  const div = d3.select(slice.selector);
+  const nodes = json.data.nodes;
+  const fd = slice.formData;
+  const f = d3.format(fd.y_axis_format);
+
+  const width = slice.width();
```

Comment:
The slice object exposes `width()` and `height()` methods that can be useful while rendering your chart.